z3-sys/Cargo.toml

```diff
@@ -17,6 +17,7 @@ repository = "https://github.com/prove-rs/z3.rs.git"
 
 [build-dependencies]
 bindgen = { version = "0.70", default-features = false, features = ["runtime"] }
+cc = "~1.1"
```

Comment:
Could you add a comment here about why this is pinned, so that we remember easier and can go back and fix it more correctly?

Reply:
Added a note in [531708c](https://github.com/prove-rs/z3.rs/pull/329/commits/531708c4d37f1dc0623f95e42eafb7b73b71ec32)